HTML/exercise-05/script.js

```diff
@@ -0,0 +1,220 @@
+var apikey = "fd717b92";
+
+var movies = [];
+var found = false;
+
+var textinput = document.getElementById("input");
+var btn = document.getElementById("Search");
+var cardCont = document.getElementsByClassName("cardCont")[0];
+var paginationDiv = document.getElementById("paginationDiv");
+var paginationP = document.getElementById("paginationP");
+var prevBtn = document.getElementById("prevBtn");
+var nextBtn = document.getElementById("nextBtn");
+var sortOrderId = document.getElementById("sortOrderId");
+var sortbyId = document.getElementById("sortbyId");
+var ifram = document.getElementById("iframe");
+var sortingDivCont = document.getElementsByClassName("sortingDivCont")[0];
+
+sortingDivCont.style.display = "none";
+paginationDiv.style.display = "none";
+var currettext = "";
+var page = 1;
+
+prevBtn.onclick = () => {
+  if (page - 1 > 0) {
+    page = page - 1;
+    getPage(page);
+  }
+};
+
+nextBtn.onclick = () => {
+  // if(page -1 > 0){
+  page = page + 1;
+  getPage(page);
+  // }
+};
+
+btn.onclick = getData;
+
+function getData() {
+  paginationDiv.style.display = "none";
+  fetch(
+    "http://www.omdbapi.com/?apikey=fd717b92&type=movie&s=" + textinput.value
```

Comment:
ideally, you'd use a env variable but here you could make a variable to store the baseURL/api key since it's used multiple times